docker-compose.yml

```diff
@@ -103,8 +110,14 @@ services:
       - "$MINION_QUEUE"
   incron:
     <<: *backend-conf
-    # This service watch for new images to trigger ocr and robotoff processing
+    # This service watches for new images to trigger ocr and robotoff processing
     command: ["perl", "scripts/run_cloud_vision_ocr.pl", "/mnt/podata/new_images"]
+  redis-listener:
+    <<: *backend-conf
+    # we want the redis_listerner for off, but not for pro profile
+    profiles: ["off"]
+    # This service watches for new events on a redis stream to delete users
+    command: ["perl", "scripts/listen_to_redis_stream.pl"]
```

Comment:
@john-gom this means we have to define a new systemd service in prod to do this ?

Reply:
Hi @alexgarel . Yes, I had created an issue in the project here: https://github.com/orgs/openfoodfacts/projects/119?pane=issue&itemId=89376373&issue=openfoodfacts%7Copenfoodfacts-server%7C11082